test/conftest.py

```diff
@@ -89,16 +89,18 @@ def etcd_session(request, tmp_path_factory):
     tmpdir = tmp_path_factory.mktemp("etcd")
     host = "localhost"
     port = 12388
-    etcd_instance = EtcdInstance(host, port, tmpdir)
+    etcd_instance = etcd_helper.EtcdInstance(host, port, tmpdir,
+                                             username=etcd_helper.etcd_username,
+                                             password=etcd_helper.etcd_password)
```

Comment:
The naming a little bit confusing here. I suggest to rename the public constants:
```suggestion
                                             username=etcd_helper.DEFAULT_USERNAME,
                                             password=etcd_helper.DEFAULT_PASSWORD)
```

Reply:
Renamed to the public constants and use it as default values in etcd class.